### PR TITLE
Rules for strawpoll.me

### PR DESCRIPTION
**Pull request by fuglede**

### CODE REVIEW

File: https-everywhere-checker [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 2ec94f1ce11a2046408a8df228447e2f0dd85cda
+Subproject commit ce35ca84ec75f3205b098d5de73ea293fc22f187
```
Comment (reedy) [inline https-everywhere-checker]: Did you mean to update this submodule?

Comment (fuglede) [inline https-everywhere-checker]: Nope, didn't actually touch the file, so I assumed that one of the tests had done it.

Comment (reedy) [inline https-everywhere-checker]: Looks like you're essentially reverting https://github.com/EFForg/https-everywhere/commit/7b76e745566cff08c9638b0b7138e2d0d35199b6

Might want to git pull, then run git submodule update --init to make sure your locally checked out submodule is fully up to date. Not sure if you want to just then add a commit to this patch chain to revert this change, or resubmit the pull request

Comment (fuglede) [inline https-everywhere-checker]: Doesn't appear to do all that much:
```bash
$ git submodule update --init
Submodule 'addon-sdk' () registered for path 'addon-sdk'
Submodule 'https-everywhere-checker' () registered for path 'https-everywhere-checker'

$ git commit -a
# On branch strawpoll-rules2
nothing to commit (working directory clean)
```

Comment (fuglede) [inline https-everywhere-checker]: Yeah, not sure exactly why, but doing the same thing on a (fresh) master seemed to do the trick, so I included this in #1169 instead. Thanks.
